src/d3a/models/strategy/area_agents/two_sided_pay_as_clear_engine.py

```diff
@@ -161,7 +161,7 @@ def _match_offers_bids(self):
             self._delete_forwarded_offer_entries(offer)
 
     def tick(self, *, area):
-        super().tick(area=area)
+        # super().tick(area=area)
```

Comment:
Please remove

Reply:
You dont need the whole function then, right?

Reply:
@spyrostz I removed it, as it was calling two_sided_pay_as_bid_engines event_tick which doesn't makes sense to me. Do you have any opinion on that?

Reply:
@faizan2590 yes, some form of it should be here, because if you remove it completely the IAA will not forward the offers. The problem is that it is problematic as it is right now, since the way it is implemented, it forwards the bids twice, and it also performs market clearings for both two sided pay as clear AND pay as bid at the same time. Therefore this needs to change to not call super().tick() but the one sided engine constructor, bypassing the two sided pay as bid constructor. Even better would be to handle this via function overloading of course, which would remove the need of bypassing the constructor.